src/components/config/parse-har.ts

```diff
@@ -25,6 +25,33 @@
 import { Config } from '../../interfaces/config'
 import { RequestConfig } from '../../interfaces/request'
 
+const convertNameValueArraysToObject = (
+  headers: {
+    name: string
+    value: any
+  }[]
+) => {
+  const obj: any = {}
+  headers.forEach((item) => {
+    if (item.name.charAt(0) !== ':') {
+      obj[item.name] = item.value
+    }
+  })
+  return obj
+}
+
+const parsePostData = (postData: any) => {
+  if (!postData) {
+    return {}
+  }
+
+  if (postData.mimeType === 'application/json') {
```

Comment:
Do we need to prepare `multipart/form-data` mime type too?

Reply:
yeah, but let's support json first for now.